src/charts/bar.js

```diff
@@ -43,8 +44,21 @@ export default function Bar (_container) {
   }
 
   const MIN_BAR_HEIGHT = 1
+  const DIMMED_COLOR = "silver"
 
-  const getColor = (d) => scales.colorScale(d[keys.ID])
+  const getColor = (d) => {
+    const key = d.data && d.data.key
+
+    if (key && config.selectedKeys.length && Array.isArray(config.selectedKeys)) {
```

Comment:
@biovisualize this type checking for `config.selectedKeys` makes me think we may want to do some kind of validation on the config object in `chart.js` as to avoid doing it here at the module / component level.

Reply:
I would recommend reversing the checks `Array.isArray` and `.length`; as the doing `.length` first will throw an error if `selectedKeys` is `undefined`, where as if you did `Array.isArray` check first it would short circuit without error

Reply:
That's a good idea. A validator could live in a pipe with the autoconfig.

Reply:
I'll make an issue for it so we can track it